packages/notte-core/src/notte_core/credentials/bitwarden.py

```
@@ -0,0 +1,156 @@
+            )
+        if not self._access_token:
+            raise ValueError("Bitwarden access token required. Set BWS_ACCESS_TOKEN env var or pass access_token=")
+        self._secrets_cache = self._fetch_secrets()
+        logger.info(f"[BitwardenVault] Loaded {len(self._secrets_cache)} secrets")
+
+    @override
+    def stop(self) -> None:
+        self._secrets_cache = None
+
+    def _run_bws(self, *args: str) -> str:
+        cmd = [self._bws_path] + list(args) + ["--output", "json"]
```

Ruff (0.15.12)

[warning] Consider `[self._bws_path, *list(args), "--output", "json"]` instead of concatenation

Replace with `[self._bws_path, *list(args), "--output", "json"]`

(RUF005)